### PR TITLE
cached_property moved namespace

### PR DESCRIPTION
**Pull request by carlbordum**
Hi, this namespace for `cached_property` was deprecated.

### CODE REVIEW

**Comment by vicmunoz** [comment]
+1

File: flask_testing/utils.py [hunk 1]
```diff
@@ -32,7 +32,11 @@
     # Python 2 urlparse fallback
     from urlparse import urlparse, urljoin
 
-from werkzeug import cached_property
+try:
+    from werkzeug.utils import cached_property
+except ImportError:
```
Comment (davidism) [inline flask_testing/utils.py]: The `try/except` isn't necessary, `cached_property` has always been available from `werkzeug.utils`.

Comment (carlbordum) [reply flask_testing/utils.py]: Thanks. Fixed.

**Comment by niciqy** [comment]
When do we expect this to be merged?

**Comment by filipallberg** [comment]
It would be nice to have this PR or #145 merged.

**Comment by springrid** [comment]
@jcomo @jarus ?

**Comment by davidism** [comment]
Stop pinging people or adding +1 comments, it won't get it done faster and clogs up everyone else's notifications.

**Comment by victorbergelin** [comment]
Hi @jcomo @jarus, could we have this merged?

**Comment by jcomo** [comment]
@carlbordum could you rebase this against master so that we can see passing tests?

**Comment by carlbordum** [comment]
@jcomo I rebased and my commit became irrelevant thanks to the much more comprehensive changes by @jnozsc.

Closed thanks to https://github.com/jarus/flask-testing/pull/145

**Pull request closed without merging**